### ISSUE TITLE
Export or print to PDF

### ISSUE THREAD

**Issue by humphreylee** [helix-editor/helix#6287]
There is a [pandoc ](https://github.com/helix-editor/helix/issues/1541) way to export to PDF, but I wonder if an integrated approach can be implemented to provide complete solution, like how other functionalities being baked into Helix, like typical modern GUI editor. Thanks for all the hard work.

**Comment by pascalkuthe** [comment]
I don't think we would want to include that in core. There is no reason to do this inside helix instead of using an external command (be it oandoc or something else). It doesn't integrate with editing in any specucal way. It's just kind of text related.

While we do include more feature in core than something like vim. The goal is not to include anything that might be useful as some GUI programs do but primarily functionality that actually enhance the text editing experience (like lsp) or closely interacts with it.

**Comment by gabydd** [comment]
Agree with everything Pascal said, this should be a plugin (or maybe if the build server protocol gets implemented it could be used for this), anyways I think this should probably be converted to a discussion as it is not an actionable issue.
